packages/yarnpkg-builder/sources/tools/makeConfig.ts

```diff
@@ -79,6 +79,19 @@ export const makeConfig = (config: webpack.Configuration): webpack.Configuration
   },
 
   plugins: [
+    process.env.BUILD_MONITORING_ENABLED && new (require(`@datadog/build-plugin/dist/webpack`).BuildPlugin)({
```

Comment:
Might be worth using something like:

```ts
function getBuildPlugin() {
  if (!test)
    return [];
  const BuildPlugin = require('...')
  return [new BuildPlugin()];
}
plugins: [
  ...getBuildPlugins(),
  new OtherPlugin1(),
  new OtherPlugin2(),
]
```

This way we don't need to type or filter the array